appscale/tools/appscale_tools.py

```diff
@@ -51,6 +57,12 @@ def async_layout_upgrade(ip, keyname, script, error_bucket, verbose=False):
     error_bucket.put(ssh_error)
 
 
+MIN_FREE_DISK_DB = 40.0
```

Comment:
I would recommend putting these thresholds on the server side so that we can present consistent warnings on both the dashboard and the tools. If we ever decide to change the value, it will be easier if it's just defined in one spot. We may also want to allow the user to configure these thresholds eventually.

I'm not sure about the best way to implement it, though. One idea would be to include a "warnings" section in the stats passed by the controller. Unfortunately, those aren't technically stats, so I'm not sure if that's the cleanest way.

Another possibility would be to add a new controller endpoint to retrieve current thresholds. However, that has the downside of requiring fixed categories. For example, if we wanted to add something like `MAX_IOPS` (just as a contrived example), we would need to change to tools so that they could make sense of the new category.

Do you have any ideas about how those warnings should be communicated?